src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java

```diff
@@ -127,566 +130,196 @@ public static Response performRequest(RestClient client, Request request) {
     }
 
     public static String getAccountIndexMapping() {
-        return "{  \"mappings\": {" +
-                " \"properties\": {\n" +
-                "          \"gender\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"fielddata\": true\n" +
-                "          }," +
-                "          \"address\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"fielddata\": true\n" +
-                "          }," +
-                "          \"firstname\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"fielddata\": true,\n" +
-                "            \"fields\": {\n" +
-                "              \"keyword\": {\n" +
-                "                \"type\": \"keyword\",\n" +
-                "                \"ignore_above\": 256\n" +
-                "              }" +
-                "            }" +
-                "          }," +
-                "          \"lastname\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"fielddata\": true,\n" +
-                "            \"fields\": {\n" +
-                "              \"keyword\": {\n" +
-                "                \"type\": \"keyword\",\n" +
-                "                \"ignore_above\": 256\n" +
-                "              }" +
-                "            }" +
-                "          }," +
-                "          \"state\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"fielddata\": true,\n" +
-                "            \"fields\": {\n" +
-                "              \"keyword\": {\n" +
-                "                \"type\": \"keyword\",\n" +
-                "                \"ignore_above\": 256\n" +
-                "              }" +
-                "            }" +
-                "          }" +
-                "       }"+
-                "   }" +
-                "}";
+        String mappingFile = "mappings/account_index_mapping.json";
+        URL url = Resources.getResource(mappingFile);
+        try {
+            return Resources.toString(url, Charsets.UTF_8);
+        } catch(Exception e) {
+            return null;
+        }
     }
 
     public static String getPhraseIndexMapping() {
-        return "{  \"mappings\": {" +
-                " \"properties\": {\n" +
-                "          \"phrase\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"store\": true\n" +
-                "          }" +
-                "       }"+
-                "   }" +
-                "}";
+        String mappingFile = "mappings/phrase_index_mapping.json";
```

Comment:
all the method have almost same implementation, the only different is mappingFile name, could you simply it?

Reply:
Thank you for the review! I made a function to make logic and error handling common.
Also found `fileToString` method in TestUtils, so i stopped using Resources and used that method.